src/traffic_manager/traffic_manager.cc

```diff
@@ -708,9 +710,18 @@ main(int argc, const char **argv)
 
   RecRegisterStatInt(RECT_NODE, "proxy.node.config.draining", 0, RECP_NON_PERSISTENT);
 
-  const int MAX_SLEEP_S      = 60; // Max sleep duration
-  int sleep_time             = 0;  // sleep_time given in sec
-  uint64_t last_start_epoc_s = 0;  // latest start attempt in seconds since epoc
+  int sleep_time             = 0; // sleep_time given in sec
+  uint64_t last_start_epoc_s = 0; // latest start attempt in seconds since epoc
+
+  std::random_device rd;
+  std::mt19937 gen(rd());
+  std::uniform_int_distribution<> dis(0, 999);
```

Comment:
The variance should be larger than 1 second.

Reply:
If the variance is larger than the starting delay, you can get very odd sequences of wait times. It would be possible to have it wait 11s, then 2s, then 14s, then 10s, then 26s. That seems a bit off to me.

Reply:
Yeah, I talked to @brbzull0 about it and recommend having a variable 0% to 50% of the sleep time.

Other recommendations can be made...